web-console/4.0-designs/code-editors/code-editor.md

```diff
@@ -0,0 +1,63 @@
+# Code Editor
+
+## Edit YAML
+
+![pic 1](img/1.png)
+
+- The code area is borderless at the top and has the button bar at the bottom as a separator.
+- The Edit action can be removed from the Actions dropdown on every detail page. Users can access the tab as needed.
+- The Download and Expand links are sticky in the top right corner of the YAML editor, with a vertical separator between them. The icons belong to the left of each link.
```

Comment:
I know you mentioned it below, but since the "view samples" link is also shown in this image, maybe it should be called out and explained after download and expand?